src/components/menu/menu.tsx

```diff
@@ -65,6 +66,12 @@ export class Menu {
     @Prop({ reflect: true })
     public badgeIcons = false;
 
+    /**
+     * Renders list items in a grid layout, rather than a vertical list
+     */
+    @Prop({ reflect: true })
+    public gridLayout = false;
```

Comment:
By the way, there is one thing I still want to fix in the grid layout of menus.
It's about the separator lines (`<li class="mdc-deprecated-list-divider" role="separator"></li>`).

I want them to span through an entire row, when menu content is rendered in a grid. Currently, they will be just like others and take a cell.

Not a merge blocker, but if I manage to fix it before merge, I'll add a fixup here.

Reply:
done ⚡️ and pushed fixups